src/services/userManager.ts

```diff
@@ -0,0 +1,36 @@
+import { User } from 'pnc-api-types-ts';
+
+import * as userService from 'services/userService';
+
+/**
+ * Class managing information about user.
+ */
+class UserManager {
+  // null means user information is not available
+  private user: User | null = null;
+
+  public removeUser() {
```

Comment:
I would remove this one, if it's not used or there is no specific already existing use case